signingscript/docker.d/passwords.yml

```diff
@@ -165,8 +165,8 @@ in:
              "cas_new_webextensions_rsa"
           ]
           - ["https://autograph-external.prod.autograph.services.mozaws.net",
-             {"$eval": "AUTOGRAPH_FENIX_USERNAME"},
-             {"$eval": "AUTOGRAPH_FENIX_PASSWORD"},
+             {"$eval": "AUTOGRAPH_FENNEC_USERNAME"},
+             {"$eval": "AUTOGRAPH_FENNEC_PASSWORD"},
```

Comment:
I don't see these variables used anywhere in `dev` or `fake-prod`. I'm not sure we should use them.